update framework with edge coupled SiN PCM

mustafacc opened the pull request.

mustafacc committed:
update framework with edge coupled SiN PCM

mustafacc committed:
delete stray edge coupler

lukasc-ubc commented:
The design is on Layer 1/5.  Could you move to 4/0?

<img width="241" alt="image" src="https://github.com/user-attachments/assets/f17b1763-54e1-44f7-a8aa-a0c3927a48c2">

lukasc-ubc commented:
Also we may want to put the origin in the middle, like ANT requests.

lukasc-ubc commented:
also, please remove the FloorPlan box (the one that covers the entire chip), since I will use that to check for overlaps with other Floorplans before placing the cell.

lukasc-ubc committed:
move_layer 1/5->4/0, Floorplan-based merge

Use the Floorplan layer when merging, to ensure designs don't overlap with each other. Should work for any PCM shapes.

lukasc-ubc commented:
I remap the layers to 4/0 in the EBeam_merge.py script.
